terror/terror.go

```diff
@@ -44,6 +45,12 @@ const (
 	CodeMissConnectionID ErrCode = iota + 1
 )
 
+// Special error codes.
+const (
+	// CodeResultUndetermined indicates the sql execution result is undetermined.
+	CodeResultUndetermined ErrCode = iota + 1
```

Comment:
Does _CodeResultUndetermined_ has the same value compare to _CodeMissConnectionID_?